uninstall.sh

```diff
@@ -0,0 +1,291 @@
+  rm -rf /etc/pterodactyl /usr/local/bin/wings /var/lib/pterodactyl
+}
+
+rm_services() {
+  systemctl disable --now mariadb
+  systemctl disable --now pteroq
+  rm -rf /etc/systemd/system/pteroq.service
+  case "$OS" in
+  debian | ubuntu)
+    systemctl disable --now redis-server
+    ;;
+  centos)
+    systemctl disable --now redis
+    systemctl disable --now php-fpm
+    rm -rf /etc/php-fpm.d/www-pterodactyl.conf
+    ;;
+  esac
+}
+
+rm_cron() {
+  # shellcheck disable=SC2063
+  crontab -l | grep -v "* * * * * php /var/www/pterodactyl/artisan schedule:run >> /dev/null 2>&1" | crontab -
+}
+
+rm_database() {
+  valid_db=$(mysql -u root -p -e "SELECT schema_name FROM information_schema.schemata;" | grep -v -E -- 'schema_name|information_schema|performance_schema')
+  warning "Be careful! This database will be deleted!"
+  if [[ "$valid_db" == *"panel"* ]]; then
+    echo -n "* Database called panel has been detected. Is it the pterodactyl database? (y/N): "
+    read -r is_panel
+    [[ "$is_panel" =~ [Yy] ]] && DATABASE=panel || echo "$valid_db"
+  else
+    echo "$valid_db"
+  fi
+  while [ -z "$DATABASE" ] || [[ $valid_db != *"$database_input"* ]]; do
+    echo -n "* Choose the panel database(to skip type none): "
+    read -r database_input
+    DATABASE=$database_input
+  done
+  [[ "$DATABASE" != "none" ]] && mysql -u root -p -e "DROP $DATABASE"
+  # Exclude usernames User and root (Hope no one uses username User)
+  valid_users=$(mysql -u root -p -e "SELECT user FROM mysql.user;" | grep -v -E -- 'User|root')
+  warning "Be careful! This user will be deleted!"
+  if [[ "$valid_users" == *"pteroactyl"* ]]; then
+    echo -n "* User called pterodactyl has been detected. Is it the pterodactyl user? (y/N): "
+    read -r is_user
+    [[ "$is_user" =~ [Yy] ]] && USER=pterodactyl || echo "$valid_users"
+  else
+    echo "$valid_users"
+  fi
+  while [ -z "$USER" ] || [[ $valid_users != *"$user_input"* ]]; do
+    echo -n "* Choose the panel user(to skip type none): "
+    read -r user_input
+    USER=$user_input
+  done
+  [[ "$USER" != "none" ]] && mysql -u root -p -e "DROP USER '$USER'@'127.0.0.1'"
+  mysql -u root -p -e "FLUSH PRIVILEGES;"
+}
+
+## MAIN FUNCTIONS ##
+
+perform_uninstall() {
+  [ "$RM_PANEL" == true ] && rm_panel_files
+  [ "$RM_PANEL" == true ] && rm_services
+  [ "$RM_PANEL" == true ] && rm_cron
+  [ "$RM_PANEL" == true ] && rm_database
+  [ "$RM_WINGS" == true ] && rm_wings_files
+}
+
+main() {
+  rm_database
+  exit 1
+  detect_distro
+  print_brake 70
+  output "Pterodactyl uninstallation script"
+  output
+  output "Copyright (C) 2018 - 2021, Vilhelm Prytz, <vilhelm@prytznet.se>"
+  output "https://github.com/vilhelmprytz/pterodactyl-installer"
+  output
+  output "Sponsoring/Donations: https://github.com/vilhelmprytz/pterodactyl-installer?sponsor=1"
+  output "This script is not associated with the official Pterodactyl Project."
+  output
+  output "Running $OS version $OS_VER."
+  print_brake 70
+  # check_os_comp
```

Comment:
Why is this commented?

Reply:
Testing purposes :) thanks for catching.